source/loader/layers/sanitizer/linux/backtrace.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+#include "stacktrace.hpp"
```

Comment:
we have something similar already that does not need to invoke a process - https://github.com/oneapi-src/unified-runtime/blob/main/source/loader/layers/validation/backtrace.hpp. Can you use that instead?

Reply:
In fact, I reused some of this code. I added "llvm-symbolizer" because this code is not able to provide enough debug information. For example, if I compile sycl code with "-g", what I get from it is 

```
./a.out() [0x403eee]
```

but "llvm-symoblizer" will give

```
main
../test-e2e/AddressSanitizer/double-free/double-free-1.cpp:24:3
```

This is why we need "llvm-symoblizer" or other similar tools.

Reply:
Hi @pbalcer, should I force a dependency on libbacktrace? 
If so, should we add `FetchContent` in cmake or third_party in UR repo?

Reply:
If you need human-readable traces by default, then yes, we need to make this library a dependency. Libbacktrace uses makefiles, so `FetchContent` is probably not going to work. This probably needs to use https://cmake.org/cmake/help/v3.17/module/ExternalProject.html.

Reply:
I also noticed that it's not a cmake project. 
Thank for your help, I'll try it.

> If you need human-readable traces by default, then yes, we need to make this library a dependency.

I agree that it is not a required dependency, but personally, making it a required dependency will make our tool more user-friendly. I'll try to enable it just for path finding, and if someone disagrees with making it as required, I can modify this later.